pyaedt/edb_core/components.py

```diff
@@ -1522,6 +1522,72 @@ def update_rlc_from_bom(
                 self.components[comp].is_enabled = False
         return found
 
+    @pyaedt_function_handler()
+    def import_bom(
+        self,
+        bom_file,
+        delimiter=",",
+        refdes_col=0,
+        part_name_col=1,
+        comp_type_col=2,
+        value_col=3,
+    ):
+        """Load external BOM file.
+
+        Parameters
+        ----------
+        bom_file : str
+            Full path to the BOM file, which is a delimited text file.
+        delimiter : str, optional
+            Value to use for the delimiter. The default is ``","``.
+        refdes_col : int, optional
+            Column index of refdes. The default is ``"0"``.
+        part_name_col : int, optional
+             Column index of part name. The default is ``"1"``. Set to None if not exist.
+        comp_type_col : int, optional
+            Column index of component type. The default is ``"2"``.
+        value_col : int, optional
+            Column index of value. The default is ``"3"``. Set to None if not exist.
+        Returns
+        -------
+        bool
+        """
+        with open(bom_file, "r") as f:
+            lines = f.readlines()
+            unmount_comp_list = list(self.components.keys())
+            for l in lines[1:]:
+                print(l)
```

Comment:
I would suggest to remove the `print`. If necessary, can we use the `logger` instead?
```suggestion
```